src/index.js

```diff
@@ -140,41 +153,44 @@ module.exports = {
     'no-useless-concat': 'error',
     'no-var': 'error',
     'no-void': 'error',
-    'no-warning-comments': 'off',
-    'no-with': 'off',
-    'object-curly-spacing': 'off',
+    'no-warning-comments': 'error',
+    'no-with': 'error',
+    'object-curly-spacing': ['error', 'always'],
     'object-shorthand': 'error',
-    'one-var': 'off',
+    'one-var': ['error', 'never'],
     'operator-assignment': 'error',
-    'operator-linebreak': 'off',
-    'padded-blocks': 'off',
+    'operator-linebreak': 'error',
+    'padded-blocks': ['error', 'never'],
     'prefer-arrow-callback': 'error',
     'prefer-const': 'error',
     'prefer-spread': 'error',
     'prefer-template': 'error',
-    quotes: 'off',
-    'quote-props': 'off',
+    'quote-props': ['error', 'as-needed'],
+    quotes: ['error', 'single'],
     radix: 'error',
     'require-yield': 'error',
-    semi: 'off',
-    'semi-spacing': 'off',
-    'sort-vars': 'error',
-    'space-before-blocks': 'off',
-    'space-before-function-paren': 'off',
-    'space-in-parens': 'off',
-    'space-infix-ops': 'off',
-    'space-unary-ops': 'off',
-    'spaced-comment': 'off',
+    semi: 'error',
+    'semi-spacing': 'error',
     'sort-imports': ['error', {
-      'ignoreCase': false,
-      'ignoreMemberSort': false,
-      'memberSyntaxSortOrder': ['none', 'all', 'single', 'multiple']
+      ignoreCase: false,
+      ignoreMemberSort: false,
+      memberSyntaxSortOrder: ['none', 'all', 'single', 'multiple']
+    }],
+    'sort-vars': 'error',
+    'space-before-blocks': 'error',
+    'space-before-function-paren': ['error', 'never'],
+    'space-in-parens': 'error',
+    'space-infix-ops': 'error',
+    'space-unary-ops': ['error', {
+      nonwords: false,
+      words: false
```

Comment:
The jscs rule only seems to apply to nonwords. If we apply to words also, that means writing code like `typeof!foo` and `new[foo][0]`, which seems to go against our convention. Is this intentional?

Otherwise, if we don't want this rule to apply to words, we'll be using the rule's default values, and thus we can simplify these lines to `'space-unary-ops': 'error',`